src/utils/ExtensionLoader.js

```diff
@@ -393,27 +394,54 @@ define(function (require, exports, module) {
         
         // Load *subset* of the usual builtin extensions list, and don't try to find any user/dev extensions
         if (brackets.inBrowser) {
-            var basePath = PathUtils.directory(window.location.href) + "extensions/default/",
-                defaultExtensions = [
-                    // Core extensions we want to support in the browser
-                    "CSSCodeHints",
-                    "HTMLCodeHints",
-                    "JavaScriptCodeHints",
-                    "SVGCodeHints",
-                    "HtmlEntityCodeHints",
-                    "InlineColorEditor",
-                    "JavaScriptQuickEdit",
-                    "QuickOpenCSS",
-                    "QuickOpenHTML",
-                    "QuickOpenJavaScript",
-                    "QuickView",
-                    "UrlCodeHints",
-
-                    // Custom extensions we want loaded by default
-                    // "HTMLHinter",
-                    "brackets-browser-livedev",
-                    "brackets-paste-and-indent"
-                ];
+            var basePath = PathUtils.directory(window.location.href) + "extensions/default/";
+
+            // We have a set of defaults we load if not instructed to do
+            // otherwise via query string params.
+            var defaultExtensions = [
+                "CSSCodeHints",
+                "HTMLCodeHints",
+                "JavaScriptCodeHints",
+                "SVGCodeHints",
+                "HtmlEntityCodeHints",
+                "InlineColorEditor",
+                "JavaScriptQuickEdit",
+                "QuickOpenCSS",
+                "QuickOpenHTML",
+                "QuickOpenJavaScript",
+                "QuickView",
+                "UrlCodeHints",
+
+                // Custom extensions we want loaded by default
+                // "HTMLHinter",
+                "brackets-browser-livedev",
+                "brackets-paste-and-indent"
+            ];
+
+            // Add any extra extensions we found on qs extraExtensions param
+            if (params.get("extraExtensions")) {
+                var extraExtensions = params.get("extraExtensions").split(/\s*\,\s*/);
```

Comment:
Could documenting the format you're expecting on each parameter be worthwhile? How would you feel about something like:

```js
// We expect a comma-delimited string of extension names
var delimiter = /\s*\,\s*/;

// ...
if (params.get("extraExtensions")) {
  var extraExtensions = params.get("extraExtensions").split(delimiter);
  // ...
}
```